### PR TITLE
Add modelers blog view count API

### PR DESCRIPTION
**Pull request by G-Sacker**

### CODE REVIEW

**Comment by shishupei** [comment]
开始门禁检查，等门禁检查通过后可以合入

File: src/main/java/com/datastat/dao/QueryDao.java [hunk 1]
```diff
@@ -3946,4 +3946,37 @@ public String getViewCount(CustomPropertiesConfig queryConf, String repoType, St
...
+        try {
+            for (var bucket : buckets) {
+                var jsonObject = objectMapper.createObjectNode();
+                jsonObject.put("id", bucket.get("key").get("id").asText());
+                jsonObject.put("title", bucket.get("key").get("title").asText());
+                jsonObject.put("count", bucket.get("doc_count").asInt());
+                resJsonArray.add(jsonObject);
+            }
+        } catch (Exception e) {
+            logger.error("query/modelers/blog/view get error", e.getMessage());
```
Comment (Kaede10) [inline src/main/java/com/datastat/dao/QueryDao.java:3977]: logger.error格式错误

**Review by Kaede10 — COMMENTED** [review]
加UT测试用例

**Comment by shishupei** [comment]
<table style="border-collapse: collapse" border=1>
    <tr>
        <th>检查项</th>
        <th>状态</th>
        <th>日志</th>
    </tr>

<tr>
    <td>DT覆盖率</td>
    <td>2.29%</td>
    <td><a href="https://opensourceways-ci.test.osinfra.cn/log/datastat-server/301/301_dt覆盖率.html">查看日志</a></td>
</tr>

<tr>
    <td>Check代码检查</td>
    <td>&#9989;</td>
    <td><a href="https://opensourceways-ci.test.osinfra.cn/log/datastat-server/301/301_Check代码检查.html">查看日志</a></td>
</tr>

<tr>
    <td>敏感信息扫描</td>
    <td>&#9989;</td>
    <td><a href="https://opensourceways-ci.test.osinfra.cn/log/datastat-server/301/301_敏感信息扫描.html">查看日志</a></td>
</tr>

<tr>
    <td>安全编码扫描</td>
    <td>&#9989;</td>
    <td><a href="https://opensourceways-ci.test.osinfra.cn/log/datastat-server/301/301_安全编码扫描.html">查看日志</a></td>
</tr>

<tr>
    <td>开源代码漏洞扫描</td>
    <td>&#10060;</td>
    <td><a href="https://opensourceways-ci.test.osinfra.cn/log/datastat-server/301/301_开源代码漏洞扫描.html">查看日志</a></td>
</tr>

<tr>
    <td>流水线链接</td>
    <td colspan="2"><a href="https://devcloud.cn-north-4.huaweicloud.com/cicd/project/3a76c1785dda4b13a399937d1978f240/pipeline/detail/66fb93d75f8d41f0a1082bbf4770f5a9/0f42c3bca9684198ad00d847a74149cd">点击跳转</a></td>
</tr>
</table>

**Pull request closed without merging**